src/NRediSearch/Client.cs

```diff
@@ -421,52 +479,95 @@ public async Task<bool> AddHashAsync(RedisKey docId, double score, bool replace)
         /// <summary>
         /// Get the index info, including memory consumption and other statistics.
         /// </summary>
-        /// <remarks>TODO: Make a class for easier access to the index properties</remarks>
         /// <returns>a map of key/value pairs</returns>
-        public Dictionary<string, RedisValue> GetInfo() =>
-            ParseGetInfo(DbSync.Execute("FT.INFO", _boxedIndexName));
+        public InfoResult GetInfo() =>
```

Comment:
I think we should leave the old API alone here (breaking changes, etc), and add a new API for the parsed version; perhaps `GetInfo()` and `GetInfoParsed()` ?